src/Config/Converters/StringJsonConverterFactory.cs

```diff
@@ -44,8 +48,7 @@ class StringJsonConverter : JsonConverter<string>
             if (reader.TokenType == JsonTokenType.String)
             {
                 string? value = reader.GetString();
-
-                return Regex.Replace(reader.GetString()!, ENV_PATTERN, new MatchEvaluator(ReplaceMatchWithEnvVariable));
+                return Regex.Replace(value!, ENV_PATTERN, new MatchEvaluator(ReplaceMatchWithEnvVariable));
```

Comment:
value is nothing but reader.GetString() which is already stored in the previous line. We don't need to get the string again from the reader - its gonna give us the same 'value'. reader doesn't advance unless we go out of this function. Also, its guaranteed to be non-null since we already checked that the TokenType is String.